src/main/fc/settings.c

```diff
@@ -653,6 +653,21 @@ const clivalue_t valueTable[] = {
     { "pid_in_tlm",                 VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = {TABLE_OFF_ON }, PG_TELEMETRY_CONFIG, offsetof(telemetryConfig_t, pidValuesAsTelemetry) },
 #if defined(TELEMETRY_IBUS)
     { "report_cell_voltage",        VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_TELEMETRY_CONFIG, offsetof(telemetryConfig_t, report_cell_voltage) },
+    { "ibus_sensor1",               VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 255 }, PG_TELEMETRY_CONFIG, offsetof(telemetryConfig_t, flysky_sensors[0])},
```

Comment:
What are these used for?
Also, if they really should be an array, and not individually named, then they should be accessed as an `ARRAY` in CLI.

Reply:
@mikeller 
Thanks, just seen syntax. This is the list of supported telemetry sensors, at once up to 15 sensors can be configured.

Reply:
Ok. Waiting for more updates to this then. There is not much point in mergeing this in its incomplete state.

Reply:
@mikeller, okey, I have working version but I need to add them step by step.

Reply:
Ok, just add the steps to this pull request then.